tests/unittests/EntryPoint.cpp

```diff
@@ -15,21 +15,27 @@
 //******************************************************************************
 
 #include <windows.h>
-
 #include <TestFramework.h>
 
 int main(int argc, char** argv) {
+// ::CoInitialize, Uninitialize, don't exist on OSX, aren't needed
+#ifdef WIN32
     // Initialize COM for all of the tests
     ::CoInitializeEx(NULL, COINIT_MULTITHREADED);
 
     testing::InitGoogleTest(&argc, argv);
-
     LOG_INFO("Starting unit tests...");
-
     auto result = RUN_ALL_TESTS();
 
     // TODO: move to a scopeguard when we have one in our codebase
     ::CoUninitialize();
 
+#else
+    testing::InitGoogleTest(&argc, argv);
+    printf("Starting unit tests...\n");
+    auto result = RUN_ALL_TESTS();
```

Comment:
Seems out of scope for this change? I'd end up having to leave at least part of it ifdef'd anyway, and I'm not sure I fully understand all the implications of this TODO.

---
In reply to: [72163061](https://github.com/Microsoft/WinObjC/pull/634#discussion_r72163061) [](ancestors = 72163061)